src/txorphanage.cpp

```diff
@@ -241,3 +241,19 @@ void TxOrphanage::EraseForBlock(const CBlock& block)
         LogPrint(BCLog::TXPACKAGES, "Erased %d orphan tx included or conflicted by block\n", nErased);
     }
 }
+
+std::vector<CTransactionRef> TxOrphanage::GetChildren(const CTransactionRef& parent, NodeId peer) const
```

Comment:
In f4d8fe713a036c4d2c1e7c2329077c34f75c8608: It seems `NodeId peer` is not used in `GetChildren`.